Index.html

```diff
@@ -14,38 +14,36 @@
 <body>
     <header class="toolbar">
         <a class="logo">Welcome</a>
-        <nav class="hamburguer"><img src="./Images/Union.png" alt="hamburguer menu"></nav>
-        
+        <nav class="hamburguer"><img src="./Images/Union.png" alt="hamburguer menu"></nav>        
     </header>
```

Comment:
- [ ] The Item Navigation should be at the center. Kindly fix it as you don't have a navigation Item on the desktop screen.
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/172987997-b759994f-7857-4b1f-9279-503e420b1806.png)